examples/rustapi_module/tests/test_datetime.py

```diff
@@ -40,21 +40,8 @@ def tzname(self, dt):
 MAX_MICROSECONDS = int(pdt.timedelta.max.total_seconds() * 1e6)
 MIN_MICROSECONDS = int(pdt.timedelta.min.total_seconds() * 1e6)
 
-IS_X86 = platform.architecture()[0] == "32bit"
-IS_WINDOWS = sys.platform == "win32"
-if IS_WINDOWS:
-    MIN_DATETIME_FROM_TIMESTAMP = pdt.datetime.fromtimestamp(86400)
-    if IS_X86:
-        MAX_DATETIME_FROM_TIMESTAMP = pdt.datetime.fromtimestamp(32536789199)
-    else:
-        MAX_DATETIME_FROM_TIMESTAMP = pdt.datetime.fromtimestamp(32536799999)
-else:
-    if IS_X86:
-        MIN_DATETIME_FROM_TIMESTAMP = pdt.datetime.fromtimestamp(-2147483648)
-        MAX_DATETIME_FROM_TIMESTAMP = pdt.datetime.fromtimestamp(2147483647)
-    else:
-        MIN_DATETIME_FROM_TIMESTAMP = pdt.datetime.fromtimestamp(-62135510400)
-        MAX_DATETIME_FROM_TIMESTAMP = pdt.datetime.fromtimestamp(253402300799)
+MIN_DATETIME = pdt.datetime(1970, 1, 1)
```

Comment:
I agree with using `pdt.datetime` literals, but I disagree with both of these choices, they are reducing the effectiveness of the tests by arbitrarily removing a lot of the interesting edge cases!

I think we should start by switching the current conditional boundaries over to using datetime literals instead of `fromtimestamp`, then possibly (probably in a second PR), add some additional tests that run the datetime-related tests in different time zone locales.

Reply:
I agree this would be much better. But I think removing the edge cases until then would still be an improvement since I fear that at the moment any errors these tests will find will be ignored because of its flakiness e.g. https://ci.appveyor.com/project/fafhrd91/pyo3/builds/28239822/job/dnquxyqc6sratqlk. Also at the moment they are causing a bit of inconvenience by permanently failing locally for several developers.

Reply:
Here is the new version (I have tested it, it does not cause the failures):

```python
if IS_WINDOWS:
    MIN_DATETIME_FROM_TIMESTAMP = pdt.datetime(1970, 1, 2, 0, 0)
    if IS_X86:
        MAX_DATETIME_FROM_TIMESTAMP = pdt.datetime(3001, 1, 19, 4, 59, 59)
    else:
        MAX_DATETIME_FROM_TIMESTAMP = pdt.datetime(3001, 1, 19, 7, 59, 59)
else:
    if IS_X86:
        # TS ±2147483648 (2**31)
        MIN_DATETIME_FROM_TIMESTAMP = pdt.datetime(1901, 12, 13, 20, 45, 52)
        MAX_DATETIME_FROM_TIMESTAMP = pdt.datetime(2038, 1, 19, 3, 14, 8)
    else:
        MIN_DATETIME_FROM_TIMESTAMP = pdt.datetime(1, 1, 2, 0, 0)
        MAX_DATETIME_FROM_TIMESTAMP = pdt.datetime(9999, 12, 31, 18, 59, 59)
```

Reply:
Thanks, I changed it as suggested.